.github/workflows/main.yaml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    container:
+      image: google/dart:latest
+
+    strategy:
+      matrix:
+        package: ['angular_bloc', 'bloc', 'bloc_test', 'flutter_bloc']
+      fail-fast: true
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
```

Comment:
why do we need `setup-java`?

Reply:
https://github.com/subosito/flutter-action/issues/15 this says it's somehow broken otherwise? dunno, we might try to remove it.

Reply:
If it's not too much trouble can you try without it? Seems weird to me 🤔

Reply:
Will try.